packages/client/hmi-client/src/components/workflow/ops/funman/funman-operation.ts

```diff
@@ -3,21 +3,35 @@ import type { FunmanInterval, TimeSpan } from '@/types/Types';
 
 const DOCUMENTATION_URL = 'https://github.com/siftech/funman';
 
-export interface ConstraintGroup {
-	borderColour: string;
-	name: string;
+export enum ConstraintType {
+	State = 'state variable(s)',
+	Parameter = 'parameter(s)',
+	Observable = 'observable(s)'
+}
 
-	// One of
-	// - monotonicityConstraint
-	// - stateConstraint
-	constraintType: string;
+export enum DerivativeType {
+	LessThan = 'less than',
+	GreaterThan = 'greater than',
+	Increasing = 'increasing',
+	Decreasing = 'decreasing',
+	LinearlyConstrained = 'linearly constrained',
+	Following = 'following'
+}
```

Comment:
Your terminologies are confusing here, as only 2 out of 6 items here have something to do with derivatives. 

I would recommend these changes to better reflect the vocabulary/semantic at hand cc @liunelson 
- ConstraintType => Constraint
- DerivativeType => ConstraintType